### DISCUSSION TITLE
How to center text align on cell?

### DISCUSSION THREAD

**Discussion by idafoh** [glideapps/glide-data-grid#347] [Q&A] [answered]
I want to center text align cell content. But I cannot find any option in `themeOverride` type?

**Comment by jassmith** [comment, marked as answer]
4.1.0 beta has a contentAlign property which does what you want.

**Reply by chunyukuo88** [reply]
I have been trying to use this but it does not seem to be a part of the code anymore... was this removed before version `^5.0.1-alpha2`? I also noticed that none of the content in the stories have any cells where the text is centered:

https://glideapps.github.io/glide-data-grid/?path=/story/tests-testcases--grid-no-trailing-blank-row

If `contentAlign` has indeed been removed, how do I center text in cells? If it is still usable, can you give an example? I performed a pretty exhaustive search in the API docs and this repo and didn't find any mention of it.

Thank you

**Reply by romanstetsyk** [reply]
It is there in version 5.2.1.

**Reply by chunyukuo88** [reply]
Thanks for your response. Do you have a URL, or is an update to the documentation forthcoming? I could find instances of `contentAlign` for things like text-cell.tsx etc. but nothing for centering cells via the `DataGrid` component.

**Reply by romanstetsyk** [reply]
It's documented [here](https://github.com/glideapps/glide-data-grid/blob/main/packages/core/API.md#gridcell). It has to be specified in `getCellContent`.

**Reply by chunyukuo88** [reply]
Thank you very much, that indeed got it working.

**Comment by jassmith** [comment]
4.1.0 beta has the option you need.
